### DISCUSSION TITLE
Version 0.7.2 - Fix CDI error "Cannot manage CDI with something else than two sessions."

### DISCUSSION THREAD

**Discussion by Louhike** [Louhike/Ugly-GDEmu-Games-Manager#72] [General]
It's just a fix release to make the tool work with CDI files with an old version of the CDI format (the headset address was stored differently).
UGGM was incorrectly saying that it "Cannot manage CDI with something else than two sessions.", even though there was the correct number of sessions.

The other change with this release is that I will now include binaries which does not require the .NET Framework to be installed. The only setback is that the size is far bigger.

<hr /><em>This discussion was created from the release <a href='https://github.com/Louhike/Ugly-GDEmu-Games-Manager/releases/tag/Version_0_7_2'>Version 0.7.2 - Fix CDI error "Cannot manage CDI with something else than two sessions."</a>.</em>

**Comment by stry8993** [comment]
Hey, just wanted to say that I'm eagerly anticipating this, and I'm sure quite a few others with GDEmu's will really enjoy it. So I just wanted to say thank you for your hard work and efforts on this.

**Reply by Louhike** [reply]
Do you mean that you still have the error? It should be fixed with the last release. If you still have the error, I'd love to know with which game.

**Reply by stry8993** [reply]
Oh, I'm sorry, I was just saying thank you and that I appreciate what you folks are doing, is all. No errors, not yet, personally. Sorry for the confusion.

**Reply by Louhike** [reply]
No problem. Thanks a lot, it’s really appreciated!
